client/src/lib/tests/ShortcodeSerialiser-test.js

```diff
@@ -1,5 +1,7 @@
 /* global jest, jasmine, describe, beforeEach, it, pit, expect, process */
-
+/* eslint-disable
+ import/extensions
```

Comment:
It's not clear to me why this test can't use a normal `import` statement or why we need a `dontMock` statement here.

Reply:
update,d works as an import statement